### PR TITLE
GS Fetcher

### PR DESCRIPTION
**Pull request by Acivico**
I have added the functionality of being able to download content from gs

### CODE REVIEW

File: tests/wetlab2variations_execution_nxf_gs_input.wfex.stage [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+version: 3
+workflow_config:
+  secure: false
+# All the inputs must be URLs or CURIEs from identifiers.org
+params:
+  general:
+    rawreads:
+      c-l-a-s-s: File
+      url:
+        - gs://gcp-public-data--broad-references/arrays/Broad_GWAS_supplemental_15061359_A1/Broad_GWAS_supplemental_15061359_A1.1.3.extended.csv
```
Comment (jmfernandez) [inline tests/wetlab2variations_execution_nxf_gs_input.wfex.stage:12]: Is this path the right one? Comparing it to the corresponding S3 links definition, they do not seem the same files:

https://github.com/inab/WfExS-backend/blob/02fadee6d919d30433de3a739b8f0bd8f0490d0e/tests/wetlab2variations_execution_nxf_s3_input.wfex.stage#L12

**Review by jmfernandez — CHANGES_REQUESTED** [review]
Have a look at the comments I have left in the stage file. Mainly related to the gs links not pointing to the right files, by comparing the names of the pointed resources at the S3 stage file: https://github.com/inab/WfExS-backend/blob/02fadee6d919d30433de3a739b8f0bd8f0490d0e/tests/wetlab2variations_execution_nxf_s3_input.wfex.stage#L12-L13

**Comment by Acivico** [comment]
I have already changed the url

File: tests/wetlab2variations_execution_nxf_gs_input.wfex.stage (hunk 1 above)
Comment (lrodrin) [reply tests/wetlab2variations_execution_nxf_gs_input.wfex.stage]: @Acivico I take a look at your changes applied in the 752c4e2e5979034f202bacc30d7bd26c61045073 comment and it seems there are not applied well. Can you update again the file as @jmfernandez indicates in this comment, but for gs?

> Have a look at the comments I have left in the stage file. Mainly related to the gs links not pointing to the right files, by comparing the names of the pointed resources at the S3 stage file:
> 
> https://github.com/inab/WfExS-backend/blob/02fadee6d919d30433de3a739b8f0bd8f0490d0e/tests/wetlab2variations_execution_nxf_s3_input.wfex.stage#L12-L13

**Review by jmfernandez — APPROVED** [review]

**Pull request merged**